cds_migrator_kit/records/templates/cds_migrator_kit_records/summer-student.html

```diff
@@ -0,0 +1,70 @@
+{#
+ Copyright (C) 2015-2018 CERN.
+  cds-migrator-kit is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{%- extends config.CDS_MIGRATOR_KIT_BASE_TEMPLATE %}
```

Comment:
Will we have a page per collection? Isn't it always the same template?

Reply:
this part I need to work on still, but good point, @jrcastro2 do you remember why was this template added?

Reply:
Right, we still need to get this done. I just set it up this way initially, but if we rename the template to "rdm" and update the rectype in the logger, that should do it I think.